supervisedEtl.py

```diff
@@ -198,160 +220,188 @@ def getExtErr(cat, band='i', corr=0.3):
     extErr = 1.0/f*fErr
     return extErr
 
+
 def getExtKron(cat, band='i'):
-    f = cat.get('flux.kron.'+band)
-    fP = cat.get('flux.psf.'+band)
+    f = cat.get('flux_kron_' + band)
+    fP = cat.get('flux_psf_' + band)
     ext = -2.5*np.log10(fP/f)
     return ext
 
+
 def getExtHsm(cat, band='i'):
     q, ext = sgsvm.getShape(cat, band, 'hsm')
     return ext
 
+
 def getExtHsmDeconv(cat, band='i'):
     q, ext = sgsvm.getShape(cat, band, 'hsmDeconv')
     return ext
 
+
 def getExtHsmDeconvNorm(cat, band='i'):
     q, ext = sgsvm.getShape(cat, band, 'hsmDeconv', deconvType='traceNorm')
     return ext
 
+
 def getExtHsmDeconvLinear(cat, band='i'):
     q, ext = sgsvm.getShape(cat, band, 'hsmDeconvLinear')
     return ext
 
+
 def getSnr(cat, band='i'):
     try:
-        f = cat.get('cmodel.flux.'+band)
-        fErr = cat.get('cmodel.flux.err.'+band)
+        f = cat.get('cmodel_flux_' + band)
+        fErr = cat.get('cmodel_flux_err_' + band)
         snr = f/fErr
     except KeyError:
         try:
-            snr = 2.5/np.log(10.0)/cat.get(band+'mag.cmodel.err')
+            snr = 2.5/np.log(10.0)/cat.get(band+'mag_cmodel_err')
         except KeyError:
-            snr = 2.5/np.log(10.0)/cat.get(band+'cmodel.mag.err')
+            snr = 2.5/np.log(10.0)/cat.get(band+'cmodel_mag_err')
     return snr
-    
+
+
 def getSnrPsf(cat, band='i'):
     try:
-        f = cat.get('flux.psf.'+band)
-        fErr = cat.get('flux.psf.err.'+band)
+        f = cat.get('flux_psf_' + band)
+        fErr = cat.get('flux_psf_err_' + band)
         snr = f/fErr
     except KeyError:
-        snr = 2.5/np.log(10.0)/cat.get(band+'mag.psf.err')
+        snr = 2.5/np.log(10.0)/cat.get(band + 'mag_psf_err')
     return snr
 
+
 def getSnrAp(cat, band='i'):
-    f = cat.get('flux.sinc.'+band)
-    fErr = cat.get('flux.sinc.err.'+band)
+    f = cat.get('flux_sinc_' + band)
+    fErr = cat.get('flux_sinc_err_' + band)
     snr = f/fErr
     return snr
 
+
 def getSeeing(cat, band='i'):
```

Comment:
Note this is a redefinition, but it's not one that was added on this PR so I guess it's ok.